airflow/contrib/operators/kubernetes_pod_operator.py

```diff
@@ -112,55 +113,55 @@ class KubernetesPodOperator(BaseOperator):  # pylint: disable=too-many-instance-
     """
     template_fields = ('cmds', 'arguments', 'env_vars', 'config_file')
 
+    @staticmethod
+    def create_labels_for_pod(context):
+        """
+        Generate labels for the pod s.t. we can track it in case of Operator crash
+
+        :param context:
+        :return:
+        """
+        labels = {
+            'dag_id': context['dag'].dag_id,
+            'task_id': context['task'].task_id,
+            'exec_date': context['ts']
+        }
+        # In the case of sub dags this is just useful
+        if context['dag'].parent_dag:
+            labels['parent_dag_id'] = context['dag'].parent_dag.dag_id
+        # Replace unsupported characters with dashes & trim at 63 chars (max allowed)
```

Comment:
This comment probably doesn't reflect what make_safe_label_value does anymore so
```suggestion
        # Ensure that label is valid for Kube, and if not trucate/remove invalid chars and replace with short hash.
```